src/main/java/uk/gov/justice/digital/hmpps/csr/api/controllers/ControllerAdvice.kt

```diff
@@ -29,8 +29,8 @@ class ControllerAdvice {
       .body(ErrorResponse(status = (HttpStatus.INTERNAL_SERVER_ERROR.value()), developerMessage = (e.message)))
   }
 
-  @ExceptionHandler(AccessDeniedException::class)
-  fun handleAccessDeniedException(e: AccessDeniedException?): ResponseEntity<ErrorResponse> {
+  @ExceptionHandler(org.springframework.security.access.AccessDeniedException::class)
```

Comment:
not quite following this change - why do you need to fully qualify?

Reply:
There was confusion with a kotlin built in AccessDeniedException, but can sort this with an import